debian/nnstreamer-python3.install

```diff
@@ -0,0 +1,2 @@
+/usr/lib/nnstreamer/filters/libnnstreamer_filter_python3.so
+/usr/lib/nnstreamer/filters/nnstreamer_python3.so /usr/lib/python3/dist-packages/
```

Comment:
are you installing a file at ```/usr/lib/python3/dist-packages/```? Is it allowed?

Reply:
Yes. It's allowed to install it at `/usr/lib/python3/dist-packages/`.
Also, I don't need to use setup.py because users can just import our modules by typing `import nnstreamer_python3`.
Of course, I can install this file to `/usr/lib/python3/dist-packages/${PKG_NAME}/`, but we have only one module so I am not sure which one is better.
As you said, I will add additional comments for documentation in source codes.

Reply:
Ok.